Library/Homebrew/cmd/tap-info.rb

```diff
@@ -64,6 +64,7 @@ def print_tap_info(taps)
             puts unless i.zero?
             info = "#{tap}: "
             if tap.installed?
+              info += "Installed"
               info += if (contents = tap.contents).blank?
                 "no commands/casks/formulae"
               else
```

Comment:
@willfaught below this line change:
- `contents.join(", ")` to `"\n#{contents.join(", ")}"`
- `info += ", private" if tap.private?` to `info += "\nPrivate" if tap.private?`

Reply:
[29c561e](https://github.com/Homebrew/brew/pull/17076/commits/29c561eb595a1410394342d79cf897ff1657fb44)